### ISSUE TITLE
Incorrect initialisation in tf_utils/layers.py

### ISSUE THREAD

**Issue by pukkapies** [openai/iaf#7]
In line 93 of the above file, it looks like the normalisation is performed over the `out_channels`, instead of `in_channels`. I think instead it should be:
`v_norm = tf.nn.l2_normalize(v, [0, 1, 3])`
